### PR TITLE
Upgrade Korolev version to 0.1.0

### PR DESCRIPTION
**Pull request by fomkin**

### CODE REVIEW

File: core/src/main/scala/wop/game/ai/WopSolver.scala [hunk 1]
```diff
@@ -9,7 +9,7 @@ import scala.language.implicitConversions
 import scala.math._
 
 object WopSolver {
-  val MAX_DEPTH = 8
+  val MAX_DEPTH = 3
```
Comment (ognick) [inline core/src/main/scala/wop/game/ai/WopSolver.scala:12]: it is korolev's part?

File: server/src/main/scala/wop/server/WopServer.scala [hunk 2]
```diff
@@ -3,120 +3,86 @@ package wop.server
...
+          actor ! PoisonPill
+        },
+        onMessage = {
+          case command =>
+            actor ! command
+        }
+      )
     },
     head = 'head (
       'link ('href /= "https://fonts.googleapis.com/css?family=Open+Sans", 'rel /= "stylesheet"),
```
Comment (ognick) [inline server/src/main/scala/wop/server/WopServer.scala:83]: why u don't use static resources?

File: server/src/main/scala/wop/server/actors/PlayerActor.scala [hunk 3]
```diff
@@ -40,7 +40,7 @@ class PlayerActor(matchMaking: ActorRef, callback: PlayerActor.Notification => U
   }
 
   def inGameReceive(name: String, init: EnterGame): Receive = {
-    case point@(_: Int, _: Int) =>
+    case Command.Point(point) =>
       init.game ! point
```
Comment (ognick) [inline server/src/main/scala/wop/server/actors/PlayerActor.scala:44]: I think, that better way, if game is in "select" state and player clicks in cell, we should cheked local cell and globall cell both. Becouse for each 9x9 small cell we know when in global cell it locates. And user doesn't want click twice, I mean

File: core/src/main/scala/wop/game/ai/WopSolver.scala (hunk 1 above)
Comment (fomkin) [reply core/src/main/scala/wop/game/ai/WopSolver.scala:12]: No. But 8 is too much for my server.

File: server/src/main/scala/wop/server/WopServer.scala (hunk 2 above)
Comment (fomkin) [reply server/src/main/scala/wop/server/WopServer.scala:83]: Historical reasons. Old Korolev wasn't support static resources.

File: server/src/main/scala/wop/server/actors/PlayerActor.scala (hunk 3 above)
Comment (fomkin) [reply server/src/main/scala/wop/server/actors/PlayerActor.scala:44]: Not part of this job.

**Comment by ognick** [comment]
#GLFM

**Review by ognick — APPROVED** [review]

**Pull request merged**